kubernetes/utils/create_from_yaml.py

```diff
@@ -34,6 +35,8 @@ def create_from_yaml(
     k8s_client: an ApiClient object, initialized with the client args.
     verbose: If True, print confirmation from the create action.
         Default is False.
+    namespace: string. Contains the namespace to create all
+        resources inside
```

Comment:
This behavior needs to be well-explained, since the "namespace" parameter is actually the namespace of the resources inside the yaml file if the yaml file does not contain the namespace in the metadata.

Reply:
I added some more documentation for this parameter.